custom/icds_reports/reports/awc_reports.py

```diff
@@ -1201,9 +1199,9 @@ def get_awc_report_lactating(order, reversed_order, awc_id):
     data = CcsRecordMonthlyView.objects.filter(
         awc_id=awc_id,
         lactating=1,
-        dob__gte=six_months_ago,
-    ).order_by('case_id', '-dob').distinct('case_id').values(
-        'case_id', 'person_name', 'dob', 'add', 'delivery_nature', 'institutional_delivery_in_month',
+        month__gte=six_months_ago,
```

Comment:
I still think for the lactating report we only want current month data.

Reply:
We are using the freshest record, but checking the last six months.

Reply:
right but we only want women who are lactating in the current month

Reply:
That's correct, have changed that.